### PR TITLE
Update map

### PR DESCRIPTION
**Pull request by ctada**
Relatively small change to update color. Does anything else on the overlay need to change?

### CODE REVIEW

File: js/visualizer.js [hunk 1]
```diff
@@ -663,50 +662,10 @@ function createCategoryControls(target, criteria) {
...
     .on("change", function (criterion, index) {
-      // Note: this mutates the critera object
-      criterion["weight"] = this.value;
-      updateSidebar({}, criteria);
-
-      // TODO: Regenerate hospital colors
-    })
-}
-
-function bindControls(criteria) {
```
Comment (ManickYoj) [inline js/visualizer.js]: Why delete these functions? Does their functionality exist elsewhere?

Comment (ManickYoj) [inline js/visualizer.js]: Nevermind I understand now. That being said, the color updating code isn't working for me.

Comment (ctada) [inline js/visualizer.js]: Are there any errors in the log/ can I see your screen? I can't replicate the error on my end.

**Comment by ctada** [comment]
Not the easiest to see colors change, unless weightings are shifted significantly. But okay on my end; @phiaseitz Are you seeing the same issues as @ManickYoj ?

**Comment by phiaseitz** [comment]
It all looks good to me!

**Pull request merged**